platforms/codesandbox/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "prestart": "CI=1 yarn prisma migrate up --auto-approve --experimental",
+    "preinstall": "export CI=1 && export PRISMA_TELEMETRY_INFORMATION='e2e-tests platform codesandbox preinstall'",
+    "prestart": "CI=1 && PRISMA_TELEMETRY_INFORMATION=\"e2e-tests platform codesandbox prestart\" && yarn prisma migrate up --auto-approve --experimental",
```

Comment:
I don't think this is actually working.

Reply:
The container has the same package.json and that is where `npm install` is run, so it should work https://codesandbox.io/s/j6txg?file=/package.json

Reply:
Unfortunately only the value from `.env` is being picked up: 
https://app.redash.io/prismaio/queries/469381?p_ci=%5B%22false%22%2C%22true%22%5D&p_information=codesandbox

Reply:
Removed from the file to not create wrong expectations.

Reply:
Removed from the file to not create wrong expectations.